runelite-client/src/main/java/net/runelite/client/plugins/customcursor/CustomCursor.java

```diff
@@ -36,7 +36,7 @@
 	TROUT("Trout", "cursor-trout.png"),
 	DRAGON_SCIMITAR("Dragon Scimitar", "cursor-dragon-scimitar.png"),
 	// will fall back to RS3 Gold if anything goes wrong
-	CUSTOM_IMAGE("Custom Image", "cursor-rs3-gold.png");
+	CUSTOM_IMAGE("", "cursor-rs3-gold.png");
```

Comment:
The name should still be "Custom Image". It's the filename which should be an empty string.